docs/UsersGuide/source/ContributorsGuide.rst

```diff
@@ -0,0 +1,359 @@
+* **Target subject matter experts (SMEs) among the code management team.** When possible, tag team members who are familiar with the modifications made in the PR so that the code management team can provide effective and streamlined PR reviews and approvals. Developers can tag SMEs by selecting the gear icon next to "Assignees" (under the Reviewers list) and adding the appropriate names. 
+* **Schedule a live code review** if the PR is exceptionally complex in order to brief members of the code management team on the PR either in-person or through a teleconference. Developers should indicate in the PR message that they are interested in a live code review if they believe that it would be beneficial. 
+
+
+.. _ContribStandards:
+
+Code and Configuration Standards
+===================================
+
+General
+-----------
+
+Platform-specific settings should be handled only through configuration and module files, not in code or scripts.
+
+SRW Application
+------------------
+
+Externals.cfg
+    * All externals live in a single ``Externals.cfg`` file.
+    * Externals should point only to authoritative repositories for the given code base.
+
+        * Temporary exceptions are made for a PR into the ``develop`` branch of ``ufs-srweather-app`` that is dependent on another PR (e.g., a ``regional_workflow`` PR from the same contributor). When the component PR is merged, the contributor must update the corresponding ``ufs-srweather-app`` PR with the hash of the component's authoritative repository.
+    * Only a single hash will be maintained for any given external code base. All externals should point to this static hash (not to the top of a branch). 
+    
+Build system
+    * Each component should build with CMake
+    * Each component should build with Intel compilers on official `Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ platforms and with GNU or Intel compilers on other platforms. 
+    * Each component should have a mechanism for platform independence (i.e., no hard-coded machine-specific settings outside of established environment, configuration, and module files). 
+    * Each component should build with the latest release of the `HPC-Stack <https://github.com/NOAA-EMC/hpc-stack>`__. 
+
+Module files (env files)
+    * Each component should build using the common modules located in the ``env/srw_common`` file.
+
+..
+    COMMENT: Common what?! Add update once this is clarified. 
+
+Regional Workflow
+----------------------
+The ``regional_workflow`` repository must not contain source code for compiled programs. Only scripts, configuration files, and documentation should reside in this repository. 
+
+**General Coding Standards:** 
+    * All bash scripts must explicitly be ``#!/bin/bash`` scripts. They should *not* be login-enabled.
```

Comment:
To be clear about what "not login-enabled" means, add at the end `(i.e. do not use the -l flag).`